cmd/operator/main.go

```diff
@@ -89,6 +90,8 @@ func init() {
 	if len(namespace) == 0 {
 		namespace = "default"
 	}
+
+	serviceAccount = os.Getenv("MY_POD_SERVICE_ACCOUNT")
```

Comment:
can we be more user friendly on this one? can we query k8s api directly to ask about our own account instead of passing it down through deployment spec?